redis-enterprise-mixin/panels.libsonnet

```diff
@@ -0,0 +1,469 @@
+          g.panel.barGauge.standardOptions.threshold.step.withValue(0) + g.panel.barGauge.standardOptions.threshold.step.withColor('red'),
+          g.panel.barGauge.standardOptions.threshold.step.withValue(1) + g.panel.barGauge.standardOptions.threshold.step.withColor('green'),
+        ])
+        + g.panel.barGauge.options.withDisplayMode('basic')
+        + g.panel.barGauge.options.withOrientation('horizontal')
+        + g.panel.barGauge.options.reduceOptions.withCalcs(['lastNotNull'])
+        + g.panel.barGauge.options.withShowUnfilled(true),
+
+      nodesShardsUpPanel:
+        g.panel.barGauge.new('Shards up')
+        + g.panel.barGauge.queryOptions.withTargets([
+          signals.nodes.shardUp.asTarget(),
+        ])
+        + g.panel.barGauge.panelOptions.withDescription('Displays up/down status for the shards on the selected node.')
+        + g.panel.barGauge.standardOptions.withMin(0)
+        + g.panel.barGauge.standardOptions.thresholds.withMode('absolute')
+        + g.panel.barGauge.standardOptions.thresholds.withSteps([
+          g.panel.barGauge.standardOptions.threshold.step.withValue(null) + g.panel.barGauge.standardOptions.threshold.step.withColor('transparent'),
+          g.panel.barGauge.standardOptions.threshold.step.withValue(0) + g.panel.barGauge.standardOptions.threshold.step.withColor('red'),
+          g.panel.barGauge.standardOptions.threshold.step.withValue(1) + g.panel.barGauge.standardOptions.threshold.step.withColor('green'),
+        ])
+        + g.panel.barGauge.options.withDisplayMode('basic')
+        + g.panel.barGauge.options.withOrientation('horizontal')
+        + g.panel.barGauge.options.reduceOptions.withCalcs(['lastNotNull'])
+        + g.panel.barGauge.options.withShowUnfilled(true),
+
+      nodesNodeRequestsPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Node requests',
+          targets=[signals.nodes.nodeRequests.asTarget()]
+        )
+        + g.panel.timeSeries.panelOptions.withDescription('Total endpoint request rate for the selected node.')
+        + g.panel.timeSeries.standardOptions.withUnit('none'),
+
+      nodesNodeAverageLatencyPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Node average latency',
+          targets=[signals.nodes.nodeAverageLatency.asTarget()]
+        )
+        + g.panel.timeSeries.panelOptions.withDescription('Average latency for the selected node.')
+        + g.panel.timeSeries.standardOptions.withUnit('s'),
+
+      nodesNodeMemoryUtilizationPanel:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Node memory utilization',
+          targets=[signals.nodes.nodeMemoryUtilization.asTarget()]
+        )
+        + g.panel.timeSeries.panelOptions.withDescription('Memory utilization % for the selected node.')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      nodesNodeCPUUtilizationPanel:
+        commonlib.panels.generic.timeSeries.base.new(
```

Comment:
What are your thoughts on how this would look versus the `commonlib.panels.cpu.timeSeries.base.new`? I see that the base is empty in the common lib but it has utilization by core and mode overrides.